diff --git a/src/GDArmatureDisplay.cpp b/src/GDArmatureDisplay.cpp
index c729ff2..b90feaf 100644
--- a/src/GDArmatureDisplay.cpp
+++ b/src/GDArmatureDisplay.cpp
@@ -168,7 +168,9 @@ bool GDArmatureDisplay::is_playing() const {
 
 void GDArmatureDisplay::play(const String &_animation_name, int loop) {
 	if (has_animation(_animation_name)) {
-		getAnimation()->play(_animation_name.ascii().get_data(), loop);
+		auto anim = _animation_name.ascii().name;
+		if (anim == get_current_animation()) return;
+		getAnimation()->play(anim, loop);
 	}
 }
 
