diff --git a/src/GDArmatureDisplay.h b/src/GDArmatureDisplay.h
index ce55e20..d13115b 100644
--- a/src/GDArmatureDisplay.h
+++ b/src/GDArmatureDisplay.h
@@ -120,9 +120,10 @@ public:
 
 	Dictionary get_bones();
 	GDBone2D *get_bone(const String &name);
+	void force_parent_owned() {};
 };
 
 VARIANT_ENUM_CAST(GDArmatureDisplay::AnimMode);
 VARIANT_ENUM_CAST(GDArmatureDisplay::AnimFadeOutMode);
 
-#endif // GDARMATUREDESPLAY_H
\ No newline at end of file
+#endif // GDARMATUREDESPLAY_H
diff --git a/src/GDDisplay.h b/src/GDDisplay.h
index 731e724..ab9ea09 100644
--- a/src/GDDisplay.h
+++ b/src/GDDisplay.h
@@ -3,6 +3,7 @@
 
 #include "dragonBones/DragonBonesHeaders.h"
 #include "scene/2d/node_2d.h"
+#include "scene/resources/canvas_item_material.h"
 
 DRAGONBONES_USING_NAME_SPACE;
 
@@ -76,4 +77,4 @@ public:
     }
 };
 
-#endif // GDISPLAY_H
\ No newline at end of file
+#endif // GDISPLAY_H
