diff --git a/src/luaState.cpp b/src/luaState.cpp
index 4cb95c3..4bc582f 100644
--- a/src/luaState.cpp
+++ b/src/luaState.cpp
@@ -465,7 +465,7 @@ Variant LuaState::getVariant(lua_State *state, int index) {
 	switch (type) {
 		case LUA_TSTRING: {
 			String utf8_str;
-			utf8_str.parse_utf8(lua_tostring(state, index));
+			String::utf8(lua_tostring(state, index));
 			result = utf8_str;
 			break;
 		}
@@ -553,7 +553,7 @@ Ref<LuaError> LuaState::handleError(lua_State *state, int lua_error) {
 	switch (lua_error) {
 		case LUA_ERRRUN: {
 			String utf8_str;
-			utf8_str.parse_utf8(lua_tostring(state, -1));
+			String::utf8(lua_tostring(state, -1));
 			msg += "[LUA_ERRRUN - runtime error ]\n";
 			msg += utf8_str;
 			msg += "\n";
@@ -562,7 +562,7 @@ Ref<LuaError> LuaState::handleError(lua_State *state, int lua_error) {
 		}
 		case LUA_ERRSYNTAX: {
 			String utf8_str;
-			utf8_str.parse_utf8(lua_tostring(state, -1));
+			String::utf8(lua_tostring(state, -1));
 			msg += "[LUA_ERRSYNTAX - syntax error ]\n";
 			msg += utf8_str;
 			msg += "\n";
@@ -712,7 +712,7 @@ int LuaState::luaPrint(lua_State *state) {
 				break;
 			}
 			default: {
-				it_string.parse_utf8(lua_tostring(state, n));
+				String::utf8(lua_tostring(state, n));
 				break;
 			}
 		}
