diff --git a/bullet_server.h b/bullet_server.h
index f19ee5d..79fd4bd 100644
--- a/bullet_server.h
+++ b/bullet_server.h
@@ -9,7 +9,7 @@
 #include "core/os/os.h"
 #include "scene/2d/node_2d.h"
 #include "scene/resources/world_2d.h"
-#include "servers/physics_server_2d.h"
+#include "servers/physics_2d/physics_server_2d.h"
 
 #include <vector>
 
diff --git a/bullet_type.h b/bullet_type.h
index 0e6a071..f2756a0 100644
--- a/bullet_type.h
+++ b/bullet_type.h
@@ -6,6 +6,13 @@
 #include "scene/resources/2d/shape_2d.h"
 #include "scene/resources/texture.h"
 
+#define _USE_MATH_DEFINES
+#include <cmath>
+
+#ifndef Math_PI
+#define Math_PI M_PI
+#endif
+
 class BulletType : public Resource {
 	GDCLASS(BulletType, Resource);
 
